src/__test__/integration/avm.spec.ts

```diff
@@ -72,4 +72,43 @@ describe('Avm spec', () => {
 
         await peer.stop();
     });
+
+    it('Timeout in par call', async () => {
+        // arrange
+        const peer = new FluencePeer();
+        await peer.start();
+
+        // act
+        const promise = new Promise<string[]>((resolve, reject) => {
```

Comment:
what are you testing here?

Reply:
I'm checking if the timeout had worked and the branch without timeout got executed after the on with the timeout

Reply:
done